Assignment4.java

```diff
@@ -0,0 +1,129 @@
+
+package assignment4;
+
+import java.util.Scanner;
+
+public class Assignment4 {
+
+    /**
+     * Curtis Kunkle
+     * Assignment 4
+     * This program uses methods to display the results
+     * of a multiple choice survey.
+     */
+    public static void main(String[] args) {
+        
+        //declare character for user input
+        int choice;
+        
+        //new scanner object!
+        Scanner keyboard = new Scanner(System.in);
+        
+        //Output prompt, list killer Peavey tube amps
+        print("What is the best high-gain Peavey tube Amplifier?");
+        print("1: Classic Series");
+        print("2: ValveKing");
+        print("3: 6505");
+        print("4: XXX");
+        print("5: JSX");
+        print("6: Butcher");
+        print("7: QUIT");
+        
+        //accepts the first integer of the user's input
+        choice = keyboard.nextInt();
+        
+        //error checking.  
+        while(choice != 1 && choice != 2 &&
+              choice != 3 && choice != 4 &&
+              choice != 5 && choice != 6 && choice != 7){
+         print("Invalid choice! Try again.");
+         choice = keyboard.nextInt();
+        }
+         
+        //Output results with methods!
+        classic(choice);
+        valveking(choice);
+        Sixty505(choice);
+        xxx(choice);
+        jsx(choice);
+        butcher(choice);
+        quit(choice);
```

Comment:
Methods are also for abstraction as well as reusability. In the case of abstraction we want to create a single name that represents some logic without having to look at it each time. This logic may only be used once but it makes reading nested structures easier. 

I agree strongly with @srtinkey that placing the conditions within your methods assures each condition must run every time. It also obfuscates what the method is responsible for. While it might seem pedantic. I would rewrite this to bring the conditional structure back out of the methods and just place the condition body in a method or methods.